### ISSUE TITLE
Integration with Full Site Editing

### ISSUE THREAD

**Issue by badg0003** [kylephillips/wp-nested-pages#331]
Given WordPress' push for block themes / full site editing, it'd be great if we could allow the menu (which I believe is the classic model?) to work with the corresponding block within a given template part.

**Comment by bkeplinger** [comment]
I agree. The main reason for my installing this plugin is that it will dynamically create the site navigation based on the arrangement of pages and their settings in admin. Full site editing breaks this feature. It would be a truly wonderful thing if this functionality integrated with the full site editor.

Otherwise, I love the plugin and it's simplicity in interface.
